inductiva/coastal/bathymetry.py

```diff
@@ -363,13 +363,14 @@ def plot(
               defined.
         """
 
-        logging.info("Plotting the bathymetry on a uniform grid...")
-        depths_grid, _ = self._interpolate_to_uniform_grid(
-            x_resolution,
-            y_resolution,
-            threshold_distance=threshold_distance,
-            nullable=True,
-        )
+        if not self.is_uniform_grid():
```

Comment:
Can you add a comment on why would this work? How does the bathymetry "know" that is uniform_grid?
What does that method do?